src/sas/sasgui/perspectives/fitting/models.py

```diff
@@ -320,10 +320,9 @@ def update(self):
         new_plugins = self.findModels()
         if len(new_plugins) > 0:
```

Comment:
What is new_plugins? A list?
If it's a simple list or dict you can test for len > 0 by simply doing `if new_plugins:`. Like the following:
```
a = []
if a:
    print(a)
a = [1]
if a:
    print(a)
```